pages/1.12/cli/configure/index.md

```diff
@@ -32,3 +32,70 @@ To configure a proxy for the CLI:
 *   Define `no_proxy` for domains that you don’t want to use the proxy for:
 
         export no_proxy=".mesos,.thisdcos.directory,.dcos.directory,.zk,127.0.0.1,localhost,foo.bar.com,.baz.com”
+
+# Environment variables
+
+The DC/OS CLI supports the following environment variables, which can be set dynamically.
+
+<a name="dcos-cluster"></a>
+#### `DCOS_CLUSTER` (DC/OS CLI 0.5.x and higher only)
+
+The [attached](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/) cluster. To set the attached cluster, set the variable with the command:
+
+```bash
+export DCOS_CLUSTER=<cluster_name>
+```
+
+<a name="dcos-config"></a>
+#### `DCOS_CONFIG` (DC/OS CLI 0.4.x only)
+
+The path to a DC/OS configuration file. If you put the DC/OS configuration file in `/home/jdoe/config/dcos.toml`, set the variable with the command:
+
+```bash
+export DCOS_CONFIG=/home/jdoe/config/dcos.toml
+```
+
+If you have the `DCOS_CONFIG` environment variable configured:
+
+- After conversion to the [new configuration structure](#configuration-files), `DCOS_CONFIG` is no longer honored.
+- Before you call `dcos cluster setup`, you can change the configuration pointed to by `DCOS_CONFIG` using `dcos config set`. This command prints a warning message saying the command is deprecated and recommends using `dcos cluster setup`.
```

Comment:
Replace "Before you call `dcos cluster setup`, you can change the configuration pointed to by `DCOS_CONFIG` using `dcos config set" 

with 
Before you call `dcos cluster setup`, you can change the configuration pointed to `DCOS_CONFIG` using `dcos config set`.